crux-mir/test/Test.hs

```diff
@@ -284,6 +284,6 @@ testDir oracleTest dir = do
       gen f = doesDirectoryExist (dir </> f) >>= \case
         False -> return Nothing
         True -> Just <$> testDir oracleTest (dir </> f)
-  fs <- listDirectory dir
+  fs <- sort <$> listDirectory dir
```

Comment:
Perhaps worth a comment that the `sort` is technically unnecessary, but it ensures a stable ordering of test cases regardless of the underlying order in which the OS lists the files.